src/Axe.Windows.Telemetry/ExcludedException.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Telemetry
+{
+    /// <summary>
+    /// Class to wrap Exceptions that should be excluded from the Telemetry pipeline
+    /// </summary>
+    [Serializable]
+    public class ExcludedException : Exception
+    {
+        /// <summary>
+        /// ctor <see cref="Exception"/>
+        /// </summary>
+        /// <param name="message">The message that describes the error</param>
+        public ExcludedException(string message)
```

Comment:
I don't think any of these constructors should be allowed. I think the ExcludedException should only wrap other exceptions and have a default message indicating the exceptions type. As far as I saw only one of these constructors is currently used, making the other two unnecessary.

Reply:
I added these because of FXCop whinings, but I agree that they're not needed. Only 1 ctor exists in iteration 6.

---
In reply to: [277708173](https://github.com/Microsoft/accessibility-insights-windows/pull/331#discussion_r277708173) [](ancestors = 277708173)